tests/unit/Codeception/Lib/Driver/PostgresTest.php

```diff
@@ -124,6 +124,12 @@ public function testGetEmptyArrayIfTableHasNoPrimaryKey()
         $this->assertEquals([], $this->postgres->getPrimaryKey('no_pk'));
     }
 
+    public function testLastInsertIdReturnsSequenceValueWhenNonStandardSequenceNameIsUsed()
```

Comment:
I replaced your test with a test which actually tests the change in lastInsertId.